tests/test_cli.py

```diff
@@ -158,38 +160,6 @@ async def test_publish_subscribe(broker):
     assert sub_proc.returncode == 0
 
 
-@pytest.mark.asyncio
-async def test_pub_sub_retain(broker):
```

Comment:
these tests are replaced by ones in `test_client.py` to handle various 'will' message scenarios